standalone-metastore/pom.xml

```diff
@@ -89,7 +89,7 @@
     <libthrift.version>0.16.0</libthrift.version>
     <log4j2.version>2.18.0</log4j2.version>
     <mockito-core.version>3.3.3</mockito-core.version>
-    <orc.version>1.8.3</orc.version>
+    <orc.version>1.8.5</orc.version>
```

Comment:
I wonder, do we need to be able to define a different orc.version at the standalone-metastore than in the root pom.xml? 
Can we just use the version info from the root pom.xml?

Reply:
The parent for standalone-metastore is not hive, so it can not inherit from it.
<parent>
    <groupId>org.apache</groupId>
    <artifactId>apache</artifactId>
    <version>23</version>
  </parent>